lib/hanami/commands/new/abstract.rb

```diff
@@ -90,6 +90,10 @@ def hanami_model_version
           @hanami_model_version
         end
 
+        def hanami_version
+          Hanami::VERSION.scan(/\A\d{1,2}\.\d{1,2}/).first
```

Comment:
@davydovanton Please have a look at https://github.com/hanami/hanami/pull/503/files#diff-b6e68c896b787eed14d8d92616b64789R18, there is a discrepancy between `hanami` and `hanami-model` versions. I think we should probably let `#hanami_version` to include `~>` as prefix of the version. Eg. `"~> 0.8"`.

In this way the Ruby code is in complete control of the version number and the template is logic-less, as it only interpolates the value.